server_src/runners/state.js

```diff
@@ -2675,7 +2675,8 @@ class Runner {
           );
 
           let piSupport = await this.octoPrintService.getPluginPiSupport(
-            farmPrinters[index]
+            farmPrinters[index],
+            true
```

Comment:
Did a quick test for my WLAN Prusa OP. This API endpoint responds in 50 to 100ms. So I wonder why is a retry is needed?

Look I completely understand why we need the retry mechanism somewhere, but to be frank I'd only see its use for a OP connection test to keep the connection/power state of that OP service updated. Other than that I see no point in retrying anywhere. (Didnt read your PR analysis yet, quite tired so will try to do so now haha)